emacs-live-py-mode/live-py-mode.el

```diff
@@ -170,7 +165,7 @@ aligned but will not hide the part after the narrowing."
   ;; interactivity when `debug-on-error' is active, so it is easily possible
   ;; to miss the error. And on such an error the function will be removed
   ;; from `post-command-hook' which is confusing when not noticed.
-  (when (memq this-command '(narrow-to-region next-line viper-goto-line))
+  (when (memq this-command '(narrow-to-region next-line previous-line viper-goto-line))
```

Comment:
For performance reasons this list should be as short as possible because it requires a `redisplay` for reasons explained in the comments. Why add `previous-line`? If it is just for symmetry with `next-line` then please remove `previous-line` again and if it helps add a comment below the comment for `next-line` that it is not in the list intentionally.